src/server/frontend_wayland/text_input_v1.cpp

```diff
@@ -20,7 +20,9 @@
 #include "wl_surface.h"
 #include "mir/executor.h"
 #include "mir/scene/text_input_hub.h"
-#include "mir/wayland/client.h"
+
+#define MIR_LOG_COMPONENT "TextInputV1"
+#include "mir/log.h"
```

Comment:
I don't think we need the logging but, in general, I don't think we need `MIR_LOG_COMPONENT` at a file level. It is better to use `MIR_LOG_COMPONENT_FALLBACK` at a library level.